controllers/review.controller.js

```diff
@@ -0,0 +1,71 @@
+const reviewService = require('../services/review.service');
+
+const renderAddReview = (req, res) => {
+    res.status(200).send('Add Review');
+};
+
+const renderUpdateReview = (req, res) => {
+    res.status(200).send('Update Review');
+};
+
+//addReview controller uses addReview service to create new review
+const addReview = async (req, res) => {
+    req.body.book_id = req.params.bookId;
+    try {
+        //recieves the created review document as object
+        const review = await reviewService.addReview(req.body);
+        if (review) {
+            //updates the avg rating of book using bookRatingUpdate service
+            await reviewService.bookRatingUpdate(req.body.book_id);
+        }
+        res.status(200).send(review);
+    } catch (err) {
+        res.status(500).send(err);
+    }
+};
```

Comment:
![Screenshot 2021-05-30 at 9 03 59 AM](https://user-images.githubusercontent.com/64341057/120091156-09119f80-c126-11eb-9d7b-877a1800a0ce.png)
Here, the user id should be taken from the req.body.

Reply:
For testing purpose I used dummy user

Reply:
You can see the updated response with actual user_id- 
https://documenter.getpostman.com/view/15301567/TzXzDHC8

Reply:
Umm.. I mean from where are you taking the user_id, from the request(i.e. body raw) or you are using the user_id injected by the auth middleware?

Reply:
I am taking it from request (body raw)

Reply:
Okay just do a thing, remove the user_id from the request body and see, it should automatically add the user_id from the jwt token. If it doesn't, comment down below.

Reply:
It's not adding the user_id automatically

Reply:
Did you use the auth middleware before handling add review request or not?

Reply:
I commented all the middlewares as it showing not logged in postman doc

Reply:
And if I am using the authmiddleware it shows Cookie Access token not found

Reply:
@saggy2001  Actually auth middleware inserts the user_id into the req.body. So you have to enable that.

Reply:
And you have to login by the browser to generate the access token and then add it in the postman header cookie to solve the error

Reply:
> I commented all the middlewares as it showing not logged in postman do

From where I get the access token

Reply:
I'm explaining that on discord.

Reply:
> And you have to login by the browser to generate the access token and then add it in the postman header cookie to solve the error
> 

When I am login from browser it showing wait you are not approved.Could that make any problem